src/theory/quantifiers/bv_inverter.cpp

```diff
@@ -17,6 +17,7 @@
 #include <algorithm>
 #include <stack>
 
+#include "options/quantifiers_options.h"
```

Comment:
Alphabetize the includes.

Reply:
This particular option is at the right place :) But I'll alphabetize the others.